### PR TITLE
Add initial HotJar integration

### PR DESCRIPTION
**Pull request by LucasCairns**
### Context

> Does this issue have a Trello card?

https://trello.com/c/YMeUfcLL

### Intent

> What changes are introduced by this PR that correspond to the above card?

- Add HotJar to the main template
- Suppress personal information
- Update Helm configuration to allow HotJar ID to be configurable
- Update Circle configuration to allow HotJar ID to be configurable

### Considerations

> Are there any steps required when merging/deploying this PR?

- This should be tested extensively in Development to ensure personal information is suppressed
- Circle will need `HOTJAR_ID` adding

### Checklist

- [x] This PR contains **only** changes related to the above card
- [x] Tested in Development

### CODE REVIEW

File: server/views/components/template.njk [hunk 1]
```diff
@@ -23,10 +26,31 @@
...
+          h._hjSettings = {
+            hjid: '{{hotJarId}}',
+            hjsv: 6
+          };
+          a = o.getElementsByTagName('head')[0];
+          r = o.createElement('script');
+          r.async = 1;
+          r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
+          a.appendChild(r);
+        })(window, document, 'https://static.hotjar.com/c/hotjar-', '.js?sv=');
```
Comment (andrewrlee) [inline server/views/components/template.njk:51]: Are there some kind of allow lists that this will need to be added to? presumably on the prison side?

Comment (LucasCairns) [reply server/views/components/template.njk:51]: Good point - getting confirmation on this

Comment (LucasCairns) [reply server/views/components/template.njk:51]: I've created a card to arrange this work with CTS. This code can be merged as without the configuration in Circle it won't effectively be live

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Pull request merged**